public/index.html

```diff
@@ -16,6 +16,7 @@
     <script src="https://cdn-webgl.wrld3d.com/wrldjs/addons/searchbar/latest/searchbar.js"></script>
     <script src="https://cdn-webgl.wrld3d.com/wrldjs/addons/marker_controller/latest/marker_controller.js"></script>
     <script src="https://cdn-webgl.wrld3d.com/wrldjs/addons/indoor_control/latest/indoor_control.js"></script>
+    <script src="https://cdn-webgl.wrld3d.com/wrldjs/dist/latest/wrld.js"></script>
```

Comment:
why is this here? We are installing wrld.js using npm and it is being compiled with our code by webpack.

Reply:
For some reason without that line, the button wasn't finding the setTiltDegrees function. But when I added it in it was okay again? I'm not sure

Reply:
mhm we do not need to include this script here because we add wrld as an npm package. Let's investigate what the error is instead. It is good to avoid adding code that we do not quite understand because it will eventually come bite us later.  Ping me on teams with the error.